src/Applications/NCEP_Etc/NCEP_enkf/scripts/gmao/etc/HISTAENS.rc.tmpl

```diff
@@ -52,6 +52,8 @@ COLLECTIONS: 'bkg.eta'
 #            'inst3_3d_wxme_Np'
 #            'tavg3_2d_wxme_Nx'
 # Land output
+#            'tavg1_2d_lfo_Nx+-'
```

Comment:
See comment above -- it's probably better to make the switch to the "bkg" HISTORY "lfo" collection now

Reply:
tavg1_2d_lfo_Nx+-  is  replaced by  bkg.tavg1_2d_lfo ,  so is inst1